libs/components/packages/src/schematics/migrations/migration-collection.json

```diff
@@ -30,6 +30,11 @@
       "factory": "./update-12/icons-migration/icons-migration.schematic",
       "description": "Update icons import paths to icons library and add new peer."
     },
+    "remove-angular-eslint-prefer-standalone-component": {
```

Comment:
I think the merge put this above the new `icons-size` schematic - but alphabetically should be below.

Reply:
Good call. Updated.